src/api/Automation.ts

```diff
@@ -0,0 +1,48 @@
+import automate, { Output, OutputType } from '../automation.js';
+import { Options } from '../bin.js';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { deleteJsonFiles } from '../file.util.js';
+
+type APIOptions = Omit<
+  Options,
+  'headless' | 'output' | 'watch' | 'changeInterval' | 'port'
+>;
```

Comment:
can this omitted options be used in the programmatic api, too, or no?

Reply:
This is only TS validator because they are pointless in case of Programmatic usage of RAP. However there is no runtime validation for them, so yes, they can be used.

Reply:
Got it